contracts/erc20gas/erc20gas.go

```diff
@@ -17,17 +18,35 @@ const (
 	maxGasForCreditGasFeesTransactions uint64 = 1 * n.Million
 )
 
+var (
+	// Recalculate with `cast sig 'debitGasFees(address from, uint256 value)'`
+	debitGasFeesSelector = hexutil.MustDecode("0x58cf9672")
```

Comment:
Would be good to add the `creditGasFeesSelector` here as well.

Reply:
done